server/node/copy_from.go

```diff
@@ -38,21 +38,36 @@ type CopyFrom struct {
 	Null         string
 	Stdin        bool
 	Columns      tree.NameList
+	CopyOptions  tree.CopyOptions
 }
 
 var _ vitess.Injectable = (*CopyFrom)(nil)
 var _ sql.ExecSourceRel = (*CopyFrom)(nil)
 
 // NewCopyFrom returns a new *CopyFrom.
-func NewCopyFrom(databaseName string, tableName doltdb.TableName, fileName string, stdin bool, columns tree.NameList) *CopyFrom {
+func NewCopyFrom(databaseName string, tableName doltdb.TableName, options tree.CopyOptions, fileName string, stdin bool, columns tree.NameList) *CopyFrom {
+	var delimiterChar, nullChar string
+	switch options.CopyFormat {
+	case tree.CopyFormatBinary:
+		panic("BINARY format is not supported for COPY FROM")
+	case tree.CopyFormatText:
+		delimiterChar = "\t"
+		nullChar = "\\N"
+	case tree.CopyFormatCsv:
+		delimiterChar = ","
```

Comment:
Is there no null character for CSVs?

Reply:
Nope, unfortunately not. CSVs don't have a standard way of encoding `NULL`, so the usual convention is to have two delimiters right next to each other, and then require the empty string to be encoded as a double quoted empty string. Unfortunately the go std lib csv encoder doesn't implement this convention, so we had to fork it for Dolt and Doltgres to get that behavior.